src/index.js

```diff
@@ -54,36 +59,44 @@ function determineValidID(event) {
   }
 }
 
+function loadTravelerInfo(userID) {
+  fetchRequests.checkData(userID).then(data => {
+    currentTraveler = new Traveler(data[0])
+    destinationsData = data[1]
+    currentTrips = data[2].filter(trip => trip.userID === currentTraveler.id)
```

Comment:
If you are loading the data for a specific user ID why do you need to filter the data based on ID? Seems a little confusing